### DISCUSSION TITLE
Displaying images from (google) photos?

### DISCUSSION THREAD

**Discussion by Antlersandsunbeams** [deckerst/aves#1079] [Q&A]
This is an excellent program! I want to start moving towards using it exclusively, but am confused:

Is it possible to have Aves use my photos as a source? I'm not great with UI navigation sometimes, is that not implemented? I don't usually keep edited photos on my device; is this just a habit I'll need to adjust?  Gphotos is the only hosting solution for a lot of my photography  atm, so i'm tied to them for a minute.

**Comment by deckerst** [comment]
For now and the foreseeable future Aves only supports local files on your device. 

You can upvote/subscribe to this issue: 
- #126
